demo/BERT/builder.py

```diff
@@ -554,10 +554,12 @@ def emb_layernorm(builder, network, config, weights_dict, builder_config, sequen
     output_fp16 = trt.PluginField("output_fp16", np.array([1 if config.use_fp16 else 0]).astype(np.int32), trt.PluginFieldType.INT32)
 
     use_full_mask = 0
-    # use full_mask for XMMA kernels (use fp16/int8 precision and SM version >= 75)
-    if not config.is_calib_mode and (config.use_fp16 or config.use_int8) and (sequence_length == 128 or sequence_length == 384):
-        cc = pycuda.autoinit.device.compute_capability()
-        if cc[0] * 10 + cc[1] >= 75:
+    # use full_mask for XMMA kernels when SM version >= 75(sequence 64/96/128/384 for fp16 precision, sequence 128/384 for int8 precision)
+    cc = pycuda.autoinit.device.compute_capability()
+    if not config.is_calib_mode and cc[0] * 10 + cc[1] >= 75:
+        if config.use_fp16 and (sequence_length == 64 or sequence_length == 96 or sequence_length == 128 or sequence_length == 384):
```

Comment:
slen = 64/96 fused kernel is not integrated to OSS yet?

Reply:
Sorry, removed commit for slen =64/96.